Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/CancelDeletionProcessAsOwner/HandlerTests.cs

```diff
@@ -98,12 +103,13 @@ public async Task Publishes_domain_events()
         ).MustHaveHappenedOnceExactly();
     }
 
-    private static Handler CreateHandler(IIdentitiesRepository? identitiesRepository = null, IUserContext? userContext = null, IEventBus? eventBus = null)
+    private static Handler CreateHandler(IIdentitiesRepository? identitiesRepository = null, IUserContext? userContext = null, IEventBus? eventBus = null, IPushNotificationSender? pushNotificationSender = null)
     {
-        userContext ??= A.Fake<IUserContext>();
-        identitiesRepository ??= A.Fake<IIdentitiesRepository>();
-        eventBus ??= A.Fake<IEventBus>();
+        userContext ??= A.Dummy<IUserContext>();
+        identitiesRepository ??= A.Dummy<IIdentitiesRepository>();
+        eventBus ??= A.Dummy<IEventBus>();
+        pushNotificationSender ??= A.Dummy<IPushNotificationSender>();
```

Comment:
@tnotheis I'd appreciate feedback on this. Can I keep it, or the the old version make more sense?

Reply:
Dummies definitely make more sense (if that's what you mean).